src/components/CourseInfo/CourseInfo.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import styles from './CourseInfo.module.css';
+
+import Button from '../../common/Button/Button';
+import { mockCourses } from '../../data/data';
+import getCoursesDuration from '../../helpers/getCoursesDuration';
+import getAuthorNames from '../../helpers/getAuthorNames';
+
+const CourseInfo = () => {
+  let { courseId } = useParams();
+  let title;
+  let description;
+  let duration;
+  let authors;
+  let creationDate;
+  mockCourses.forEach((element) => {
+    if (courseId == element.id) {
+      title = element.title;
+      description = element.description;
+      duration = element.duration;
+      authors = element.authors;
+      creationDate = element.creationDate;
+    }
+  });
+
+  const leftArrowUnicode = '\u1438';
+
+  return (
+    <div className={styles.courseInfoForm}>
+      <div className={styles.backToCoursesLink}>
+        <Link to={'/courses'}>
+          <Button buttonText={' ' + leftArrowUnicode + ' Back to courses'} />
+        </Link>
+      </div>
+      <h1 className={styles.title}>{title}</h1>
+      <div className={styles.formBody}>
+        <div className={styles.description}>{description}</div>
+        <div className={styles.coursesForm}>
+          <div>
+            <p className={styles.properties}>ID: </p>
+            {courseId}
+          </div>
+          <div>
+            <p className={styles.properties}>Duration: </p>
+            {getCoursesDuration(duration)}
+          </div>
+          <div>
+            <p className={styles.properties}>Created: </p>
+            {creationDate}
+          </div>
+          <div>
+            <p className={styles.authorsProperties}>Authors: </p>
+            {authors.map((element, index) => {
+              <p key={index}>{getAuthorNames(element)}</p>;
+            })}
```

Comment:
Callback in `map` should return element, in your case it is `void`. Let's update syntax with:
```suggestion
            {authors.map((element, index) => (
                 <p key={index}>{getAuthorNames(element)}</p>
             ))}
```